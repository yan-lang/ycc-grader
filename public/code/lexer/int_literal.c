123452
4531
1234123
0
-1234
-123466
-1233
-123456
-9876