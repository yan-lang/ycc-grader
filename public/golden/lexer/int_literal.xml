<tokens>
    <token>
        <index>0</index>
        <text>123452</text>
        <type>integer literal</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>123452</value>
        <line>1</line>
        <column>1</column>
        <start>0</start>
        <stop>6</stop>
    </token>
    <token>
        <index>1</index>
        <text>4531</text>
        <type>integer literal</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>4531</value>
        <line>2</line>
        <column>1</column>
        <start>7</start>
        <stop>11</stop>
    </token>
    <token>
        <index>2</index>
        <text>1234123</text>
        <type>integer literal</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>1234123</value>
        <line>3</line>
        <column>1</column>
        <start>12</start>
        <stop>19</stop>
    </token>
    <token>
        <index>3</index>
        <text>0</text>
        <type>integer literal</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>0</value>
        <line>4</line>
        <column>1</column>
        <start>20</start>
        <stop>21</stop>
    </token>
    <token>
        <index>4</index>
        <text>-</text>
        <type>minus</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>null</value>
        <line>5</line>
        <column>1</column>
        <start>22</start>
        <stop>23</stop>
    </token>
    <token>
        <index>5</index>
        <text>1234</text>
        <type>integer literal</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>1234</value>
        <line>5</line>
        <column>2</column>
        <start>23</start>
        <stop>27</stop>
    </token>
    <token>
        <index>6</index>
        <text>-</text>
        <type>minus</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>null</value>
        <line>6</line>
        <column>1</column>
        <start>28</start>
        <stop>29</stop>
    </token>
    <token>
        <index>7</index>
        <text>123466</text>
        <type>integer literal</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>123466</value>
        <line>6</line>
        <column>2</column>
        <start>29</start>
        <stop>35</stop>
    </token>
    <token>
        <index>8</index>
        <text>-</text>
        <type>minus</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>null</value>
        <line>7</line>
        <column>1</column>
        <start>36</start>
        <stop>37</stop>
    </token>
    <token>
        <index>9</index>
        <text>1233</text>
        <type>integer literal</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>1233</value>
        <line>7</line>
        <column>2</column>
        <start>37</start>
        <stop>41</stop>
    </token>
    <token>
        <index>10</index>
        <text>-</text>
        <type>minus</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>null</value>
        <line>8</line>
        <column>1</column>
        <start>42</start>
        <stop>43</stop>
    </token>
    <token>
        <index>11</index>
        <text>123456</text>
        <type>integer literal</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>123456</value>
        <line>8</line>
        <column>2</column>
        <start>43</start>
        <stop>49</stop>
    </token>
    <token>
        <index>12</index>
        <text>-</text>
        <type>minus</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>null</value>
        <line>9</line>
        <column>1</column>
        <start>50</start>
        <stop>51</stop>
    </token>
    <token>
        <index>13</index>
        <text>9876</text>
        <type>integer literal</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>9876</value>
        <line>9</line>
        <column>2</column>
        <start>51</start>
        <stop>55</stop>
    </token>
    <token>
        <index>14</index>
        <text></text>
        <type>EOF</type>
        <source>int_literal.c</source>
        <lexer>Lexer</lexer>
        <value>null</value>
        <line>9</line>
        <column>6</column>
        <start>55</start>
        <stop>55</stop>
    </token>
</tokens>